core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -41,39 +41,15 @@ object Http4sOrgPlugin extends AutoPlugin {
 
   val githubActionsSettings: Seq[Setting[_]] =
     Seq(
-      spiewakMainBranches := Seq("main"),
+      tlCiReleaseBranches := Seq("main"),
       githubWorkflowJavaVersions := List("8", "11", "17").map(JavaSpec.temurin(_)),
-      githubWorkflowEnv += ("JABBA_INDEX" -> "https://github.com/typelevel/jdk-index/raw/main/index.json"),
-      githubWorkflowBuild := Seq(
-        WorkflowStep
-          .Sbt(List("scalafmtCheckAll", "scalafmtSbtCheck"), name = Some("Check formatting")),
-        WorkflowStep.Sbt(List("headerCheckAll"), name = Some("Check headers")),
-        WorkflowStep.Sbt(List("test:compile"), name = Some("Compile")),
-        WorkflowStep.Sbt(List("mimaReportBinaryIssues"), name = Some("Check binary compatibility")),
+      githubWorkflowBuild +=
         WorkflowStep.Sbt(
-          List("unusedCompileDependenciesTest"),
-          name = Some("Check unused compile dependencies")),
-        WorkflowStep.Sbt(List("test"), name = Some("Run tests")),
-        WorkflowStep.Sbt(List("doc"), name = Some("Build docs"))
-      ),
+          List("doc"),
+          name = Some("Build docs"),
+          cond = Some("matrix.ci == '' || matrix.ci == 'ciJVM'")),
```

Comment:
So, this `cond` here is a bit annoying.

Reply:
I don't quite understand it.

Reply:
I'm trying to account for CI matrices with a JVM/JS axis here. There might be a better way to do this.

We only want to run `doc` once per platform, but we don't know if there are platforms and if so how they are setup. E.g. we might want to do `project rootJVM; doc` and `project rootJS; doc` in their appropriate jobs. But that seems hard, since not all project have `rootJVM` and `rootJS`.

So, what I did was try to run `doc` conditionally:
1. `matrix.ci == 'ciJVM'` we are running in a JVM job in a crossed matrix, so we should run `doc`
2. `matrix.ci == ''` The `ci` key is not defined, so the matrix is not crossed, and we should run `doc`

Of course, we could run this step on all the jobs, but it would be redundant and time-consuming.

Reply:
Fixing https://github.com/typelevel/sbt-typelevel/issues/61 would fix this insanity.